wcomponents-core/src/test/java/com/github/bordertech/wcomponents/render/webxml/WHeadingRenderer_Test.java

```diff
@@ -30,27 +31,27 @@ public void testRendererCorrectlyConfigured() {
 	public void testPaint() throws IOException, SAXException, XpathException {
 		final String text = "WHeading_Test.testPaint.heading";
 
-		WHeading heading = new WHeading(WHeading.TITLE, text);
+		WHeading heading = new WHeading(HeadingLevel.H1, text);
```

Comment:
As these tests are being changed to use HeadingLevel (which they should be), it highlights the unit tests for WHeading are lacking. This would be an opportunity to improve the unit tests for WHeading, particularly around the conversion of the old Heading Level Types to Heading Level.

Reply:
Added a test to WHeading_Test to test the conversion.